### PR TITLE
Results preview

### PR DESCRIPTION
**Pull request by yjose**

### CODE REVIEW

**Comment by yjose** [comment]
link => https://deploy-preview-13--stateofdev.netlify.app/

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      "name": "Haytam Zanid",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/34218324?v=4",
+      "profile": "https://blog.zhaytam.com/",
+      "contributions": [
+        "content"
+      ]
+    },
+    {
+      "login": "boredabdel",
+      "name": "Abdel SGHIOUAR",
```
Comment (boredabdel) [inline .all-contributorsrc:97]: Abdelfettah please :)

File: LICENSE [hunk 2]
```diff
@@ -1,14 +1,20 @@
-The BSD Zero Clause License (0BSD)
+The MIT License (MIT)
 
-Copyright (c) 2020 Gatsby Inc.
```
Comment (boredabdel) [inline LICENSE:3]: The MIT License says you need to have a line with Copyright and the year https://opensource.org/licenses/MIT

So Something like "Copyright 2020 The Geeks BlaBla team" as an example without ""

File: README.md [hunk 3]
```diff
@@ -1,12 +1,84 @@
...
+<h2 align="center">
+  State Of Dev in Morocco Website
+</h2>
+<hr />
 
-## 🚀 How to Contribute
+StateOfDev.ma is a survey centered around software developers in Morocco by DevC Morocco Community.
 
-To be done :) 
+we wanted to know how we can help and support each other, and overall, be able to better respond to developers’ evolving needs.
```
Comment (boredabdel) [inline README.md:18]: We

Comment (boredabdel) [inline README.md:18]: remove , after overall

Comment (boredabdel) [inline README.md:18]: remove ’ after developers

File: README.md [hunk 4]
```diff
@@ -1,12 +1,84 @@
...
+    ```sh
+    git clone git@github.com:your-username/stateofdev.ma.git
+    ```
+
+1.  **Start developing.**
+
+    Navigate into your new site’s directory and start it up.
+
+    ```sh
+    cd stateofdev.ma/
```
Comment (boredabdel) [inline README.md:35]: Are we sure the . in the folder name will not create a issue in some OS ? In Mac it works and Linux i'm quite sure it's fine. What about Windows ?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,84 @@
...
+
+    _Note: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql)._
+
+## 🧐 Want to contribute ?
+
+If you want to contribute check out the [help wanted](https://github.com/devC-Casa/stateofdev.ma/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+sort%3Aupdated-desc) issues for things that need fixing, or suggest some new features by opening new issues.
+
+## Licensing
+
+The code in this project is licensed under MIT license.
```
Comment (boredabdel) [inline README.md:52]: the MIT license

File: README.md [hunk 6]
```diff
@@ -1,12 +1,84 @@
...
+    <td align="center"><a href="https://github.com/Ismailtlem"><img src="https://avatars1.githubusercontent.com/u/34961373?v=4?s=120" width="120px;" alt=""/><br /><sub><b>Ismail Tlemçani</b></sub></a><br /><a href="#content-Ismailtlem" title="Content">🖋</a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
```
Comment (boredabdel) [inline README.md:84]: are welcome

File: results/2020/community.mdx [hunk 7]
```diff
@@ -0,0 +1,56 @@
+---
+title: Community
+position: 6
+---
+
+# 6.Community
+
+Open-source culture and sharing knowledge are gaining a lot of popularity in Morocco lately. More than 80% of respondents have at least contributed to an open-source project. Also, Around 75% are sharing their findings and learning through blogging.
```
Comment (boredabdel) [inline results/2020/community.mdx:8]: remove - in open-source

Comment (boredabdel) [reply results/2020/community.mdx:8]: Replace capital A in Around with smaller a

File: results/2020/community.mdx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+### Are you part of any local developer's community?
+
+We don't know if people misunderstood this question or that is actually what the reality is! But more than 70% claim that they're not part of any developers community. Even more troubling, because a majority of respondents claimed they are part of the Dev Circle Morocco community ¯\_(ツ)\_/¯.
+
+<Chart id="community-q-0" title={false} />
+
+### Did you ever participate in an open-source project?
+
+Moroccan developers contribute more and more to open source, with more than 50% maintaining their own projects!
```
Comment (boredabdel) [inline results/2020/community.mdx:18]: are maintaining

Comment (boredabdel) [reply results/2020/community.mdx:18]: or better more than 50% say they are maintaining their own projects.

File: results/2020/community.mdx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+### Did you have the chance to write blog posts?
+
+While 25% of respondents are not interested in blogging, three-quarters of respondents are already bloggers or trying to make their first steps.
+
+<Chart id="community-q-2" title={false} />
+
+### How many IT events did you attend in 2019/2020?
+
+More than half of the respondents attended at least one IT event in the past two years!
```
Comment (boredabdel) [inline results/2020/community.mdx:30]: I think tech event is better than IT

File: results/2020/community.mdx [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+<Chart id="community-q-3" title={false} />
+
+### How many talks did you participate in as a speaker in 2019/2020?
+
+15% of respondents have already spoken at some IT event!
```
Comment (boredabdel) [inline results/2020/community.mdx:36]: Same as above. Tech is better

File: results/2020/community.mdx (hunk 8 above)
Comment (boredabdel) [reply results/2020/community.mdx:18]: Said no say

File: results/2020/education.mdx [hunk 11]
```diff
@@ -0,0 +1,50 @@
+---
+title: Learning & Education
+position: 3
+---
+
+# 2.Education and Learning
+
+There is a lot to be said when talking about the educational system in Morocco. We asked the community about their thoughts, opinion, source of learning and potential solution. Additionally, more than 70% of all respondents confirm that English is no more an obstacle to learning new things
```
Comment (boredabdel) [inline results/2020/education.mdx:8]: English is not an obstacle to learning new things anymore.

File: results/2020/education.mdx [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+### What languages can you read/write?
+
+With no big surprise, Arabic, English and French are the top 3 languages that the majority of respondents can read/write.
+
+<Chart id="profile-q-11" title={false} />
+
+### Is English an obstacle to learning new technologies?
+
+As you probably know, English is the language number one when it comes to learning IT. However, around 30% confirm that english is an obstacle for them to learn new technologies.
```
Comment (boredabdel) [inline results/2020/education.mdx:30]: English is the number one language

File: results/2020/education.mdx (hunk 11 above)
Comment (boredabdel) [reply results/2020/education.mdx:8]: I'm not sure if this is what you are trying to say here, let me know if i didn't understand it

File: results/2020/profile.mdx [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+### What is your highest diploma ?
+
+More than half of the developers that answered this question have an equivalent of a master degree. The other represented category were people that made a 2 year training after their high school degree (BAC). Some of the respondents had though a PHD degree.
+
+<Chart id="profile-q-4" pie title={false} />
+
+### How many years have you been coding professionally?
+
+The majority of the responses were from people that are coding for less than a year. The second group of people are people that were coding from 1 to 3 years. The majority of the people that answered this survey are therefore junior developers
```
Comment (boredabdel) [inline results/2020/profile.mdx:32]: I think this "The majority of the people that answered this survey are therefore junior developers" is a wide claim as we cannot define the seniority level of a person based only on the number of years of experience. I suggest to change to something like "Based on this data we can assume a big majority of the people who answered this survey are therefore junior developers. It's however important to note that Seniority is not only based on the numbers of years of experience"

File: results/2020/tech.mdx [hunk 14]
```diff
@@ -0,0 +1,55 @@
+---
+title: Technology
+position: 5
+---
+
+## 5.Technology
+
+Javascript is the most popular and loved programming language, in front of Java, Python, and PHP.
+
+Python takes the top spot for programming languages that developers want to learn next. Going further down the list, we also see Typescript, Go and Kotlin as most wanted programming languages
```
Comment (boredabdel) [inline results/2020/tech.mdx:10]: Golang

File: results/2020/work.mdx [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+### How long have you been unemployed after graduation?
+
+More than 80% of all respondents reported that they found a job after just a couple of months of their graduation.
+
+<Chart id="work-q-4" title={false} />
+
+### Your salary range (MAD) is
+
+Given the fact that around 30% of respondents are either Students or unemployed, it explains why we see the lowest salary came in the first place. Keep in mind that this survey covers all over Morocco, salaries vary depending on cities.
```
Comment (boredabdel) [inline results/2020/work.mdx:49]: Covers the whole of Morocco

File: src/components/Home/Hero.js [hunk 16]
```diff
@@ -9,27 +9,27 @@ export const Hero = () => (
...
           </p>
           <div className="actions">
-            <a
-              className="primary"
-              href="https://tinyletter.com/geeksBlabla/"
-              target="_blank"
-            >
-              Get Notified About The Survey
+            <a className="primary" href="#overview">
+              Read Report
```
Comment (boredabdel) [inline src/components/Home/Hero.js:22]: Read the report

File: README.md (hunk 4 above)
Comment (yjose) [reply README.md:35]: should work on windows too

File: README.md (hunk 6 above)
Comment (yjose) [reply README.md:84]: this is generated automatically with `all-contributions` bot

**Comment by boredabdel** [comment]
Looks good ;)

**Comment by yjose** [comment]
Do you think we need to change questions by titles as I did for the Technology section  ??

@aboullaite @djalal @boredabdel @AbderrahimSoubaiElidrissi @Ismailtlem

**Comment by AbderrahimSoubaiElidrissi** [comment]
@yjose I prefer titles

**Comment by zHaytam** [comment]
> Do you think we need to change questions by titles as I did for the Technology section ??
> 
> @aboullaite @djalal @boredabdel @AbderrahimSoubaiElidrissi @Ismailtlem @zHaytam @ezzarghili

IMHO, titles.

**Comment by yjose** [comment]
@zHaytam can you help with that, if you have some time 🙏

**Comment by zHaytam** [comment]
> @zHaytam can you help with that, if you have some time 🙏

Of course, I'm on it.

**Comment by yjose** [comment]
Thank you guys 🙏

**Pull request merged**